### PR TITLE
Update to the latest bridge.

### PR DESCRIPTION
**Pull request by pgavlin**
And add a .NET SDK.

### CODE REVIEW

**Comment by pgavlin** [comment]
cc @mikhailshilkov

File: sdk/python/setup.py [hunk 1]
```diff
@@ -24,13 +24,14 @@ def run(self):
...
 def readme():
-    with open('README.rst') as f:
+    with open('README.md', encoding='utf-8') as f:
         return f.read()
 
 setup(name='pulumi_random',
       version='${VERSION}',
       description='A Pulumi package to safely use randomness in Pulumi programs.',
       long_description=readme(),
+      long_description_content_type='text/markdown',
```
Comment (CyrusNajmabadi) [inline sdk/python/setup.py]: what's up with this file?

Comment (pgavlin) [reply sdk/python/setup.py]: This is a consequence of updating to a newer version of the bridge, which replaces the old rst docs for Python with Markdown.

Comment (CyrusNajmabadi) [reply sdk/python/setup.py]: Gotcha.  thanks!

**Pull request merged**